packages/titanium-color-input/src/titanium-color-input.ts

```diff
@@ -0,0 +1,254 @@
+import { css, customElement, html, LitElement, property, query } from 'lit-element';
+import { TextField } from '@material/mwc-textfield';
+
+import '@material/mwc-textfield';
+
+type MwcInput<T> = T & { mdcFoundation: { setValid(): boolean }; isUiValid: boolean };
+
+/**
+ * Material design input with color display and validation.
+ *
+ * @element titanium-color-input
+ *
+ * @fires input - Fired when the textfield receives user input.
+ *
+ * @cssprop {Color} [--app-border-color=#dadce0] - Chip border color
+ */
+@customElement('titanium-color-input')
+export class TitaniumColorInput extends LitElement {
+  /**
+   *  Value for preselecting or getting the color on the input
+   */
+  @property({ type: String }) value: string | null;
```

Comment:
It would be nice to pass-through icon and helper props too.

Reply:
okay! I will do that